utils/utils.js

```diff
@@ -0,0 +1,58 @@
+const { TerminalNodeImpl } = require('antlr4/tree/Tree.js');
```

Comment:
`utils/utils.js` is confusing. If this is the default entry for this module, should be named `index.js`